modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/rel/IgniteIndexScan.java

```diff
@@ -111,11 +104,13 @@ private IgniteIndexScan(
         @Nullable List<RexNode> proj,
         @Nullable RexNode cond,
         @Nullable IndexConditions idxCond,
-        @Nullable ImmutableBitSet requiredCols
+        @Nullable ImmutableBitSet requiredCols,
+        @Nullable RelCollation collation
```

Comment:
Did you forgot to remove annotation here?

Reply:
Fixed